controller/bindings/curl/curl.ml

```diff
@@ -60,72 +50,63 @@ let exec cmd =
   Lwt.return (result, stdout, stderr)
 
 let safe_int_of_string str =
-  try
-    Some (int_of_string str)
-  with
-    Failure _ -> None
+  try Some (int_of_string str) with Failure _ -> None
 
 let http_code_marker = '|'
 
 let parse_status_code_and_body str =
   let open Base.Option in
-  Base.String.rsplit2 ~on:http_code_marker str >>= fun (body, code_str) ->
-  safe_int_of_string code_str >>= fun code ->
-  return (code, body)
+  Base.String.rsplit2 ~on:http_code_marker str
+  >>= fun (body, code_str) ->
+  safe_int_of_string code_str >>= fun code -> return (code, body)
 
 let request ?proxy ?(headers = []) ?data ?(options = []) url =
   let cmd =
-    "", (* path to Curl executable (uses PATH if empty string) *)
-    (Array.concat
-      [ [| "curl"; Uri.to_string url
-         ; "--silent"
-         ; "--show-error"
-         ; "--write-out"; Char.escaped http_code_marker ^ "%{http_code}"
-        |]
-      ; (match proxy with
-        | Some p ->
-            [| "--proxy"
-            ;  Uri.to_string p
-            ;  "--proxy-anyauth"
-            |]
-        | None -> [| |])
-      ; (headers
-          |> List.map (fun (k, v) -> [| "--header"; (k ^ ":" ^ v) |])
-          |> Array.concat)
-      ; (match data with
-        | Some d -> [| "--data"; d |]
-        | None -> [| |])
-      ; Base.List.to_array options
-      ])
+    ( ""
+    , (* path to Curl executable (uses PATH if empty string) *)
```

Comment:
Should probably move this before the comma so it stays close to the string literal it refers to.

Reply:
Yes, will do some manual fixes once we settle on the final formatting rules.